functional_testing/Openvcloud/ovc_master_hosted/OVC/a_basic/users_tests.py

```diff
@@ -24,10 +27,33 @@ def test001_authenticate_user(self):
         #. Use U1's key again to list the accounts for U1, should succeed.
 
         """
+        self.lg('%s STARTED' % self._testID)
+
+        self.lg('- create user1 with admin access ')
+        old_password = str(uuid.uuid4()).replace('-', '')[0:10]
+        user1 = self.cloudbroker_user_create(group='admin', password=old_password )
+
+        self.lg("- Authenticate U1 with POST /cloudapi/users/authenticate API,should return session key[user1_key] .")
+        user1_key = self.get_authenticated_user_api(username=user1,password=old_password)
+        self.assertTrue(user1_key)
 
-    @unittest.skip('Not Implemented')
+        self.lg("- Use U1's key to update U1's password, should succeed.")
+        new_password = str(uuid.uuid4()).replace('-', '')[0:10]
+        response = user1_key.cloudapi.users.updatePassword(oldPassword=old_password,newPassword=new_password)
+        self.assertIn("Your password has been changed.",response)
+
+        self.lg("Check that user1's password has been reset successfully.")
+        user1_key = self.get_authenticated_user_api(username=user1,password=new_password)
+        self.assertTrue(user1_key)
+        self.lg("- Use U1's key again to list the accounts for U1, should succeed.")
+        accounts_list = user1_key.cloudapi.accounts.list()
+
+        self.lg('acountlist %s' % accounts_list)
```

Comment:
no need for this line 
as in case  assertion error ```self.assertEqual(accounts_list,[]) ``` the log will show the content of accountl_ist